deploy/move-template.sh

```diff
@@ -3,7 +3,10 @@
 set -e
 # check user
 [ "$USER" = "sphinxsearch" ] || { echo "error: script has to be run with sphinxsearch" ; exit 1 ;}
+# check for sphinx.conf syntax
 indextool --checkconfig -c conf/sphinx.conf | grep "config valid"
+# check all database queries
+for file in conf/*.part; do db=$(basename $file | cut -d '.' -f 1)_dev.; python deploy/pg2sphinx_trigger.py -d ${db} -c list -s conf/sphinx.conf; done 1>/dev/null
```

Comment:
p.e. 
```
db: uvek.conf.part will result in -d uvek.
```
the trailing point will force the script to analyze each query of the database.